HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -1319,11 +1319,11 @@ namespace WPEFramework
 
 				if ( HdmiCecSink::_instance->deviceList[n].m_physicalAddr.getByteValue(0) != 0 )
 				{
-					sprintf(&routeString[length], "%s%d", "HDMI",(HdmiCecSink::_instance->deviceList[n].m_physicalAddr.getByteValue(0) - 1));
+					snprintf(&routeString[length], sizeof(routeString) - length, "%s%d", "HDMI",(HdmiCecSink::_instance->deviceList[n].m_physicalAddr.getByteValue(0) - 1));
```

Comment:
length is inited with 0 and never updated in this function. Do we really need length?

Reply:
@apatel859 you are right it is not needed. Removed length.